src/aat/java/uk/gov/hmcts/reform/em/annotation/functional/AnnotationSetScenarios.java

```diff
@@ -36,147 +41,262 @@
+        final JSONObject annotationSet = extractJSONObjectFromResponse(response);
+        final UUID newDocumentId = UUID.randomUUID();
+        annotationSet.put("documentId", newDocumentId);
 
         request
-                .body(jsonObject.toString())
-                .post("/api/annotation-sets")
+                .body(annotationSet.toString())
+                .put("/api/annotation-sets")
                 .then()
-                .statusCode(201);
+                .statusCode(200)
+                .body("id", equalTo(annotationSetId.toString()))
+                .body("documentId", is(newDocumentId.toString()))
+                .log().all();
+    }
 
-        jsonObject.remove("documentId");
-        jsonObject.put("documentId", UUID.randomUUID().toString());
+    @Test
+    public void shouldReturn400WhenUpdateAnnotationSetWithoutId() {
+        final UUID annotationSetId = UUID.randomUUID();
+        final UUID documentId = UUID.randomUUID();
+        final ValidatableResponse response = createAnnotationSet(annotationSetId, documentId);
+        final JSONObject annotationSet = extractJSONObjectFromResponse(response);
+        annotationSet.remove("id");
 
         request
-                .body(jsonObject.toString())
+                .body(annotationSet.toString())
                 .put("/api/annotation-sets")
                 .then()
-                .statusCode(200);
+                .statusCode(400)
+                .log().all();
     }
 
     @Test
-    public void testDeleteAnnotationSetSuccess() {
-        JSONObject jsonObject = new JSONObject();
-        String annotationSetId = UUID.randomUUID().toString();
+    public void shouldReturn401WhenUnAuthenticatedUserUpdateAnnotationSet() {
+        final UUID annotationSetId = UUID.randomUUID();
+        final UUID documentId = UUID.randomUUID();
+        final ValidatableResponse response = createAnnotationSet(annotationSetId, documentId);
+        final JSONObject annotationSet = extractJSONObjectFromResponse(response);
+        final UUID newDocumentId = UUID.randomUUID();
+        annotationSet.put("documentId", newDocumentId);
+
+        unAuthenticatedRequest
+                .body(annotationSet.toString())
+                .put("/api/annotation-sets")
+                .then()
+                .statusCode(401)
+                .log().all();
+    }
 
-        jsonObject.put("documentId", UUID.randomUUID().toString());
-        jsonObject.put("id", annotationSetId);
+    @Test
+    public void shouldReturn200WhenDeleteAnnotationSetById() {
+        final UUID annotationSetId = UUID.randomUUID();
+        final UUID documentId = UUID.randomUUID();
+        final ValidatableResponse response = createAnnotationSet(annotationSetId, documentId);
+        final String id = extractJSONObjectFromResponse(response).getString("id");
+        final ValidatableResponse deletedResponse = deleteAnnotationSetById(id);
+
+        deletedResponse.statusCode(200);
```

Comment:
As this API Call would not return a Body...It would be good to return an HTTP Status 204 rather than a 200...

Reply:
yes, make sense.

Reply:
as discussed, i will raise a bug